modules/installation-guide/partials/assembly_upgrading-che-using-the-cli-management-tool-in-restricted-environment.adoc

```diff
@@ -0,0 +1,35 @@
+:parent-context-of-upgrading-prod-short-using-the-cli-management-tool-in-restricted-environment: {context}
+
+[id="upgrading-{prod-id-short}-using-the-cli-management-tool-in-restricted-environment_{context}"]
+= Upgrading {prod-short} using the CLI management tool in restricted environment
+
+:context: upgrading-prod-short-using-the-cli-management-tool-in-restricted-environment
+
+This section describes how to upgrade {prod} using the CLI management tool in restricted environment. The upgrade path supports minor version update, from {prod-short} version {prod-prev-ver} to version {prod-ver}.
+
+* xref:understanding-network-connectivity-in-restricted-environments_{context}[]
+* xref:building-an-offline-plug-in-registry-image_{context}[]
+* xref:building-an-offline-devfile-registry-image_{context}[]
+* xref:pulling-images-used-by-che-to-a-private-registry_{context}[]
+* xref:upgrading-che-using-the-cli-management-tool-in-restricted-environment_{context}[]
+
+.Prerequisites
+
+* An administrative account on an instance of {platforms-name}.
+
+* A running instance version {prod-prev-ver} of {prod}, installed using the CLI management tool on the same instance of {platforms-name}, with the {prod-cli} `--installer operator` method, in the `_<{prod-namespace}>_` project. See xref:installing-che-in-a-restricted-environment.adoc[].
```

Comment:
are we specifying `--installer operator` here to force people to use chectl/crwctl ONLY for OCP 3.11? 

In the latest chectl/crwctl, the CLI will autodetect the target version of the cluster and use the correct method:
* OCP 3.11 - operator method
* OCP 4.2+ - OLM method

Perhaps instead of telling people to use `--installer operator` method, we should explain that this old solution is only for versions of OCP which do not include OLM ?

Reply:
I believe this is more a statement to exclude `helm` based deployment. 

First time I read about `olm` vs. `operator` installer. Current docs are mainly based on the `operator` vs `helm ` distinction. Maybe we'll have to adapt and make some room for `olm`?